tee-worker/service/src/worker.rs

```diff
@@ -151,7 +151,6 @@ where
 			// FIXME: This is temporary only, as block broadcasting should be moved to trusted ws server.
 			let enclave_url = enclave.url.clone();
 			let worker_api_direct = DirectWorkerApi::new(enclave.url);
-			// related issue: https://github.com/litentry/litentry-parachain/issues/1124#issuecomment-1367690264
```

Comment:
I think we still need this comment to not forget about it (I'm not sure if it's resolved yet)

Reply:
added back.